.gitignore

```diff
@@ -48,3 +48,25 @@ passphrase
 .coverage.*
 coverage.xml
 htmlcov/
+
+\.vscode/settings\.json
+
+\.pytest_cache/v/cache/
+
+\.vscode/launch\.json
+
+lib/cherrypy/test/access\.log
```

Comment:
Consider instead something like `lib/cherrypy/**/*.log` if you are wanting all of those log files to be ignored, including in nested directories

Reply:
sounds good Tom